docs/customApps.md

```diff
@@ -0,0 +1,99 @@
+# Run Kai with your Applications
+
+Konveyor AI (KAI) leverages LLMs to streamline the modernization of application source code. To learn how Kai works, run this demo <insert demo link></insert>
+
+## Pre-requisites
+
+- Install [podman](https://podman.io/)
+- VS Code
+- git
+- GEN AI Credentials
+
+To run Kai with your Applications, follow these steps
+
+## Configure Application
+
+1. Navigate to the samples folder - `cd samples`
+1. In `./samples/config.py`, add your application's info to `repos` dict.
+
+```python
+ "<app name>": [
+        "https://github.com/<path to repo>",
+        "<source branch>",
+        "<migrated/target branch>",
+    ],
+```
+
+<app name>: The name of your application.
```

Comment:
I don't think the '<app name>:' and '<path to repo>:' syntax rendered as you wanted.  

<img width="1074" alt="Screenshot 2024-05-21 at 2 19 11 PM" src="https://github.com/konveyor-ecosystem/kai/assets/206875/bee4c7d4-d0fd-4a42-9fd5-025829944637">

Reply:
fixed it. Thanks